include/fuse.h

```diff
@@ -574,6 +574,22 @@ struct fuse_operations {
 	 */
 	int (*fallocate) (const char *, int, off_t, off_t,
 			  struct fuse_file_info *);
+
+	/**
+	 * Change the permission bits of a file
+	 *
+	 * This method is called instead of the chmod() method if the
+	 * call was invoked from an fchmod() system call.
```

Comment:
Please use the same terminology as eg. the `fgetattr` and `ftruncate` docstrings.